packages/web-new/src/scripts/console.js

```diff
@@ -198,6 +196,15 @@ function enableAPIConversionWithProtocol(editorInstance) {
                 showConsoleError("Please insert a TD which uses MQTT!")
             }
         }
+
+        if (AASTab.checked === true) {
+            if (["mqtt", "mqtts", "http", "https", "coap", "modbus"].some(p => protocolSchemes.includes(p))) {
+                generateAAS(editorInstance["_domElement"].dataset.modeId, editorInstance)
+                AASView.classList.remove("hidden")
+            } else {
+                showConsoleError("Please insert a TD which uses HTTP, MQTT, CoAP or Modbus!")
```

Comment:
So this should be changed since all protocols technically work. @SergioCasCeb could you remove the error and only display `Protocols other than HTTP, MQTT and Modbus are not officially supported by AID, do not use the resulting AID in production" on the bottom of the JSON button?